webapp/src/components/workspaceSwitcher/workspaceOptions.tsx

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+
+import Select from 'react-select'
+import {CSSObject} from '@emotion/serialize'
+import {useIntl} from 'react-intl'
+
+import {getUserWorkspaceList} from '../../store/users'
+import {UserWorkspace} from '../../user'
+import {useAppSelector} from '../../store/hooks'
+
+import './workspaceOptions.scss'
+import Search from '../../widgets/icons/search'
+
+const style = {
```

Comment:
Why style here vs css?

Reply:
Missed this. They are no longer needed. Removed now.